dhis-2/dhis-support/dhis-support-cache-invalidation/src/main/java/org/hisp/dhis/cacheinvalidation/redis/CacheInvalidationEnabledCondition.java

```diff
@@ -35,22 +36,27 @@
 /**
  * @author Morten Svanæs <msvanaes@dhis2.org>
  */
-public class DebeziumCacheInvalidationEnabledCondition extends PropertiesAwareConfigurationCondition
+public class CacheInvalidationEnabledCondition extends PropertiesAwareConfigurationCondition
 {
     @Override
     public boolean matches( ConditionContext context, AnnotatedTypeMetadata metadata )
     {
-        if ( isTestRun( context ) )
+        if ( SystemUtils.isCacheInvalidationInTest( context.getEnvironment().getActiveProfiles() ) )
+        {
+            return true;
+        }
+
+        if ( SystemUtils.isTestRun( context.getEnvironment().getActiveProfiles() ) )
         {
             return false;
         }
 
-        return getConfiguration().isEnabled( ConfigurationKey.DEBEZIUM_ENABLED );
+        return getConfiguration().isEnabled( ConfigurationKey.REDIS_CACHE_INVALIDATION_ENABLED );
```

Comment:
Do we have to think about transition for installations that already used DEBEZIUM?